docs/_articles/en/index.md

```diff
@@ -3,41 +3,44 @@ title: Salesforce Code Analyzer Plug-In
 permalink: /
 lang: en
 ---
+
 <div class="slds-notify slds-notify_alert slds-theme_alert-texture slds-text-heading_small slds-theme_info" role="alert">
   <span class="slds-assistive-text">success</span>
     We're constantly improving Salesforce Code Analyzer. Tell us what you think!
     &nbsp;&nbsp;
-	<a href="https://research.net/r/SalesforceCA" target="_blank"><button class="slds-button slds-button_brand">Give Feedback</button></a>
+    <a href="https://research.net/r/SalesforceCA" target="_blank"><button class="slds-button slds-button_brand">Give Feedback</button></a>
 </div>
 <br>
+
+<!-- temporary comment until v3.x becomes the new norm -->
 <div class="slds-notify slds-notify_alert slds-theme_alert-texture slds-text-heading_small slds-theme_success" role="alert">
   <span class="slds-assistive-text">success</span>
-  	{{ site.data.versions-v3.releasedon }}: New pilot version {{ site.data.versions-v3.scanner }} of Salesforce Code Analyzer is available
+  	New version {{ site.data.versions-v3.scanner }} of the plug-in has been released on {{ site.data.versions-v3.releasedon }}
 	&nbsp;&nbsp;
 	<a href="./en/v3.x/whats-new-v3/">Check out what's new</a>
 </div>
 <br>
+
 <div class="slds-notify slds-notify_alert slds-theme_alert-texture slds-text-heading_small slds-text-align_center slds-theme_warning" role="alert">
   <span class="slds-assistive-text">warning</span>
-  	{{ site.data.versions.releasedon }}: The latest version v{{ site.data.versions.scanner }} is available&nbsp;&nbsp;<a href="./en/release-information/">Release Information</a>
+  	Salesforce Graph Engine is available for you to try it out &nbsp;&nbsp;&nbsp;<a href="./en/v3.x/salesforce-graph-engine/introduction/">Read more</a>
 </div>
 <br>
+
+<!-- uncomment this once v3 becomes latest -->
```

Comment:
Can we uncomment this now so that we don't miss updating it later? The next release should make v3 the `latest` anyway.

Reply:
I think we probably want to actually delete this block entirely. It's redundant with the other block, and I'm not sure why that one should be deleted and this one should be added.

Reply:
Agree, this does look repeatitive.